### PR TITLE
Refactor/accessibility improvement

### PR DESCRIPTION
**Pull request by Davidschlundtbodien**
# Description
Add aria labels to the icon buttons of the application and adds lang tag to the html. Lighthouse report currently at a 97 from the previous 56.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?
- [ ] open index.html
- [ ] console.log()
- [x] dev tools
- [ ] npm test

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (joanafbrito) [inline dist/index.html:2]: Great job remembering to add language

File: dist/index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
   <body>
     <header>
       <h1><span id='header-name'></span>FITLIT</h1>
-      <button type='button' name='button' id='profile-button'></button>
+      <button aria-label="profile dropdown" type='button' name='button' id='profile-button'></button>
```
Comment (joanafbrito) [inline dist/index.html:13]: I like the aria labels you have created!

**Review by joanafbrito — APPROVED** [review]
Great job remembering to add language!
I really like the aria labels you have created!

File: dist/index.html [hunk 3]
```diff
@@ -218,7 +218,7 @@ <h4 id='sleep-friend-worst-sleeper'></h4>
           </div>
         </section>
         <section class='calendar-card hide' id='sleep-calendar-card'>
-          <button type='button' name='button' class='go-back-button sleep-go-back-button'></button>
+          <button aria-label="return to sleep card" type='button' name='button' class='go-back-button sleep-go-back-button'></button>
```
Comment (jhortizu01) [inline dist/index.html:221]: Great work on labeling the buttons!

**Review by jhortizu01 — APPROVED** [review]
Great work on labeling all the buttons.

**Pull request merged**